server/middleware/searchHandler.js

```diff
@@ -70,6 +68,19 @@ async function getMemberExternalKey(memberUuid) {
     return response.data.externalKey;
 }
 
+const returnSearchWorkstackColumns = (workstackTypeColumns, request) => {
```

Comment:
This could potentially be abstracted further, so rather than specifically returning search workstack columns, you could have for example 'returnWorkstackTypeColumns(suffix)' which returns a function, then from that you can create a 'returnSearchWorkstackColumns(workstackTypeColumns, request)' and a 'returnMyCasesWorkstackColumns(workstackTypeColumns, request)' and others.

Reply:
I have made this change